plugins/catalog-backend/src/service/router.ts

```diff
@@ -38,16 +38,10 @@ export async function createRouter(
 
   if (entitiesCatalog) {
     // Entities
-    router
-      .get('/entities', async (_req, res) => {
-        const entities = await entitiesCatalog.entities();
-        res.status(200).send(entities);
-      })
-      .get('/entities/:id', async (req, res) => {
```

Comment:
is this route required and ticketed somewhere else? Or are we skipping it for now in favour of the search endpoint?

Reply:
Not required for now. When we get around to implementing the item view soon, we should add it back but with a different path (it needs to contain both kind, name and namespace).